src/components/modules/banners/digitalSubscriptions/DigitalSubscriptionsBanner.tsx

```diff
@@ -71,6 +72,7 @@ export const DigitalSubscriptionsBanner: React.FC<BannerProps> = ({
     tracking,
     submitComponentEvent,
     countryCode,
+    numArticles = 0,
```

Comment:
if we are defaulting this to 0, will it get overridden somewhere?

Reply:
Yes, it's a prop that's getting passed in from/via frontend or DCR.